pipelines/managed/fbc-release/README.md

```diff
@@ -24,6 +24,9 @@ Tekton release pipeline to interact with FBC Pipeline
 | trustedArtifactsDebug           | Flag to enable debug logging in trusted artifacts. Set to a non-empty string to enable                                             | Yes      | ""                                                        |
 | dataDir                         | The location where data will be stored                                                                                             | Yes      | /var/workdir/release                                      |
 
+## Changes in 5.1.0
```

Comment:
fyi we are in the process of removing the changelogs

Reply:
Removed changelog and leave pipeline version unchanged.